Formula/ecl.rb

```diff
@@ -12,13 +13,23 @@ class Ecl < Formula
     sha256 "f86b66f182afdc3b7a8e8e51ca3e230d6ccc308d760faae1b2b345fd1ead0ab3" => :yosemite
   end
 
+  depends_on "libatomic_ops" => :build
+  depends_on "pkg-config" => :build
+  depends_on "bdw-gc"
   depends_on "gmp"
+  depends_on "libffi"
 
   def install
     ENV.deparallelize
     system "./configure", "--prefix=#{prefix}",
                           "--enable-threads=yes",
-                          "--with-system-gmp=yes"
+                          "--enable-gmp=system",
+                          "--enable-boehm=system",
+                          "--enable-libatomic=system",
+                          "CFLAGS=#{`pkg-config --cflags libffi`.chomp}",
```

Comment:
It would be preferable to avoid the pkg-config dependency and use

```
                          "CFLAGS=-I#{Formula["libffi"].opt_inlcude}",
                          "LDFLAGS=-L#{Formula["libffi"].opt_lib}",
                          "LIBS=-lffi"
```